hmpps-kotlin-spring-boot-autoconfigure/src/main/resources/META-INF/spring/org.springframework.boot.autoconfigure.AutoConfiguration.imports

```diff
@@ -1 +1,5 @@
-uk.gov.justice.hmpps.kotlin.HmppsKotlinConfiguration
\ No newline at end of file
+uk.gov.justice.hmpps.kotlin.auth.HmppsResourceServerConfiguration
+uk.gov.justice.hmpps.kotlin.auth.HmppsReactiveResourceServerConfiguration
+uk.gov.justice.hmpps.kotlin.auth.HmppsWebClientConfiguration
+uk.gov.justice.hmpps.kotlin.auth.HmppsReactiveWebClientConfiguration
+uk.gov.justice.hmpps.kotlin.sar.HmppsSubjectAccessRequestController
```

Comment:
I was hoping to avoid a long list here but it seems a better way to exert control of each configuration class individually.